### ISSUE TITLE
Allow suppressing update check without suppressing error messages

### ISSUE THREAD

**Issue by andersk** [charliermarsh/ruff#476]
Ruff has frequent releases, which is great, but it also means that the “new version available” message frequently adds noise to the output. Another source of noise is the “No pyproject.toml found. Falling back to default configuration…” message. A linter that prints expected noise can drown out unexpected errors, especially when it’s run from a script alongside many other linters.

```
No pyproject.toml found.
Falling back to default configuration...
Found 1 error(s).
test.py:1:5: W292 No newline at end of file

A new version of ruff is available: v0.0.82 -> v0.0.83
Run to update: pip3 install --upgrade ruff
```

Currently the only ways to suppress the update check are

* at compile time, by disabling the `update-informer` feature—not an option for those installing binaries;
* at runtime, with the `-q`/`--quiet` option—not an option for those who want to see the actual diagnostics about the input code.

There’s no way to suppress the pyproject.toml message.

Can we have an option to print *nothing* unless there are diagnostics about the input code? Perhaps this should be what `-q` means, and the existing `-q` should be `-qq` or `-s`/`--silent`?

**Comment by charliermarsh** [comment]
Agree with all of this. I think `-q` and `-s` make sense.
